policyengine_us/tests/policy/baseline/gov/states/ca/cdss/tanf/income/ca_tanf_countable_income_recipient.yaml

```diff
@@ -0,0 +1,29 @@
+- name: $1,000 earned, $500 disabli disability-based unearned, and $1,000 other unearned.
+  period: 2023
+  input:
+    state_code: CA
+    ca_tanf_earned_income: 3_000
+    ca_tanf_db_unearned_income: 500
+    ca_tanf_unearned_income: 1_000
+  output:
+    ca_tanf_countable_income_recipient: 2_450
```

Comment:
```suggestion
- name: Recipient countable income is the sum of (earned income minus $600 flat amount, times disregard percent of 50%), disability unearned income minus $600 flat amount, and total other unearned income.
  period: 2023
  input:
    state_code: CA
    ca_tanf_earned_income: 610
    ca_tanf_db_unearned_income: 601
    ca_tanf_unearned_income: 1_000
  output:
    ca_tanf_countable_income_recipient: (610-600)*.5 + (601-600) + 1_000
```

Reply:
Disability unearned income and earned income together minus the $600 flat disregard. Reference: 

> The NNI is calculated by disregarding the first $600 of disability-based unearned income and/or any earned income and 50 percent of any remaining earned income.